src/spectrecoff/Tree.fs

```diff
@@ -0,0 +1,57 @@
+[<AutoOpen>]
+module SpectreCoff.Tree
+
+open Spectre.Console
+open SpectreCoff.Layout
+open SpectreCoff.Output
+
+type GuideStyle =
+    | Ascii
+    | SingleLine
+    | DoubleLine
+    | BoldLine
+
+type TreeLayout =
+    {  Sizing: SizingBehaviour
+       Guides: GuideStyle }
+
+let defaultTreeLayout: TreeLayout =
+    {  Sizing = Collapse
+       Guides = SingleLine }
```

Comment:
Spectre also supports a "Style" method for colors, maybe it could be added as well?

Reply:
I'll look at that in a follow-up, when I add the documentation as well.